rbx_types/src/unique_id.rs

```diff
@@ -68,14 +81,30 @@ impl UniqueId {
         })
     }
 
+    /// Returns whether this `UniqueId` is 'nil' or not. That is, whether
+    /// every field of the UniqueId is set to `0`.
+    pub fn is_nil(&self) -> bool {
+        self.time != 0 && self.index != 0 && self.random != 0
```

Comment:
Fix this oopsie and then LGTM
```suggestion
        self.time == 0 && self.index == 0 && self.random == 0
```